### PR TITLE
QoL: Draw circle gizmo at the bottom of BillBoards

### PR DESCRIPTION
**Pull request by jonoser**
I was watching a stream by Cliffworms where he was using the world data editor to place items and I thought it might be helpful to have a bounding circle displayed at the bottom of billboards to make it easier to see how big they are. 

I tried to match the existing code style as closely as possible so hopefully it's not too bad.
image

### CODE REVIEW

File: Assets/Scripts/Internal/DaggerfallBillboard.cs [hunk 1]
```diff
@@ -439,6 +439,27 @@ public void OnDrawGizmos()
...
+        
+        /// <summary>
+        /// Draws a circle at the bottom of the bilboard to make it easier to judge the size regardless of rotation.
+        /// </summary>
+        public void OnDrawGizmosSelected()
+        {
+            if (!Application.isPlaying)
+            {
+                UnityEditor.SceneView sceneView = GetActiveSceneView();
+                if (sceneView)
```
Comment (andrew-raphael-lukasik) [inline Assets/Scripts/Internal/DaggerfallBillboard.cs:451]: Hi. You don't need to test `sceneView!=null`, it's UnityEditor's responsibility to decide on that when `OnDrawGizmos` is called.

File: Assets/Scripts/Internal/DaggerfallBillboard.cs [hunk 2]
```diff
@@ -439,6 +439,27 @@ public void OnDrawGizmos()
                 }
             }
         }
+        
+        /// <summary>
+        /// Draws a circle at the bottom of the bilboard to make it easier to judge the size regardless of rotation.
+        /// </summary>
+        public void OnDrawGizmosSelected()
```
Comment (andrew-raphael-lukasik) [inline Assets/Scripts/Internal/DaggerfallBillboard.cs:446]: `OnDrawGizmosSelected` is meant to be private (there are exceptions ofc, but justified)

File: Assets/Scripts/Internal/DaggerfallBillboard.cs [hunk 3]
```diff
@@ -439,6 +439,27 @@ public void OnDrawGizmos()
...
+                UnityEditor.SceneView sceneView = GetActiveSceneView();
+                if (sceneView)
+                {
+                    float radius = 0.0f;
+                    Vector3 offset = Vector3.zero;
+
+                    radius = (summary.Size.x / 2);
+                    offset.y = (summary.Size.y / 2);
+
+                    Handles.DrawWireDisc(transform.position - offset, Vector3.up, radius);
```
Comment (andrew-raphael-lukasik) [inline Assets/Scripts/Internal/DaggerfallBillboard.cs:459]: consider more concise form:
```
var sizeHalf = summary.Size * 0.5f;
UnityEditor.Handles.DrawWireDisc(transform.position - new Vector3(0, sizeHalf.y, 0), Vector3.up, sizeHalf.x);
```

File: Assets/Scripts/Internal/DaggerfallBillboard.cs (hunk 1 above)
Comment (andrew-raphael-lukasik) [reply Assets/Scripts/Internal/DaggerfallBillboard.cs:451]: Oh, ok - I just looked into the the code before your change and saw that this test was there before. I still think this line is counter-productive, no reasons for it and can be deleted (might be a legacy code, from much older editor version maybe).

File: Assets/Scripts/Internal/DaggerfallBillboard.cs (hunk 3 above)
Comment (jonoser) [reply Assets/Scripts/Internal/DaggerfallBillboard.cs:459]: Thanks for the comment, to be perfectly honest, this was basically what I started with but I started second guessing myself so I tried to make it match the existing OnDrawGizmos function to try to lessen the likelihood of it getting rejected.  I'll fix it up in the morning, Though at this point me rewriting it almost seems like a formality.

**Comment by KABoissonneault** [comment]
Can you add screenshots to show the difference?

File: Assets/Scripts/Internal/DaggerfallBillboard.cs (hunk 1 above)
Comment (jonoser) [reply Assets/Scripts/Internal/DaggerfallBillboard.cs]: Removed.

File: Assets/Scripts/Internal/DaggerfallBillboard.cs (hunk 3 above)
Comment (jonoser) [reply Assets/Scripts/Internal/DaggerfallBillboard.cs]: Updated.

**Review by KABoissonneault — APPROVED** [review]

**Review by andrew-raphael-lukasik — APPROVED** [review]

**Comment by Interkarma** [comment]
Nice addition. Cheers! :)
